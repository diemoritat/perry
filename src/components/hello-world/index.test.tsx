/* eslint-env node, jest */

import { h } from "preact";
import { render } from "preact-render-to-string";

import HelloWorld from "./";

describe("HelloWorld Snapshot", () => {
  it("should render header with content", () => {
    const tree = render(<HelloWorld />);
    expect(tree).toMatchSnapshot();
  });
});
