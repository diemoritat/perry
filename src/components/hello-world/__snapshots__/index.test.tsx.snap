// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld Snapshot should render header with content 1`] = `"<div><h1>Hello, World!</h1></div>"`;
